diff -urN splashy-0.3.2.org/src/Makefile.am splashy-0.3.2/src/Makefile.am
--- splashy-0.3.2.org/src/Makefile.am	2007-02-13 15:26:01.000000000 +0100
+++ splashy-0.3.2/src/Makefile.am	2007-05-20 00:51:12.495521697 +0200
@@ -44,7 +44,7 @@
 splashy_SOURCES = splashy_main.c splashy_functions.c
 splashy_LDADD = libsplashycnf.la libsplashy.la -lpthread -lm  -lglib-2.0
 #splashy_LDFLAGS = -static
-splashy_LDFLAGS = -all-static $(shell directfb-config  --libs --input=keyboard --imageprovider=png --font=ft2,default) $(DFBHACK)
+splashy_LDFLAGS = -all-static $(shell directfb-config  --libs --input=keyboard --imageprovider=png --font=ft2,default) -lsysfs $(DFBHACK)
 
 
 splashy_config_SOURCES = splashy_config-main.c splashy_config-functions.c \
